ykrt/src/compile/jitc_yk/codegen/x64/rev_analyse.rs

```diff
@@ -109,7 +133,7 @@ impl<'a> RevAnalyse<'a> {
                     .downcast::<X64CompiledTrace>()
                     .unwrap();
                 let vlocs = ctr.entry_vars();
-                // Side-traces don't have a trace body since we don't apply loop peeling and thus use
+                // Connector traces don't have a trace body since we don't apply loop peeling and thus use
```

Comment:
When did we rename side-traces to connector traces? We are still using `TraceKind::Sidetrace` though it seems?

Reply:
This is the fix :)

Reply:
Oohh, right! :-)